### PR TITLE
Improve border-radius preview

### LINKED ISSUE #5530: Border radius editor view and preview mismatch
## Bug Description

Border radius, view in editor and in preview looking very different. 

## Expected Behaviour

Editor and preview should appear the same

## Screenshots
image6

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## To-do

- [x] Refactor a bit
~~- [ ] Fix FE Preview, too~~
~~- [ ] Karma with snapshot~~

## Testing Instructions
- Add some elements with border and border-radius, e.g. with value 50 for visibility.
- Verify that the page carousel preview below the page and grid view look similar to the editor view.

---

### CODE REVIEW

**Comment by miina** [comment]
@barklund I still need to see if I can refactor this a bit (e.g. create helpers for the preview-values) but this cleans the code up a bit more + should improve the preview.

PTAL if you have a moment so I can finish it tomorrow first thing.

(Its base branch is #5542 since it doesn't make sense to fix it for other border types).

File: assets/src/edit-story/elements/media/editCropMoveable.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function EditCropMoveable({
 
   // Refresh moveables to ensure that the selection rect is always correct.
   useEffect(() => {
-    moveableRef.current.updateRect();
+    moveableRef.current && moveableRef.current.updateRect();
```
Comment (barklund) [inline assets/src/edit-story/elements/media/editCropMoveable.js]: Isn't this just `moveableRef.current?.updateRect()`?

File: assets/src/edit-story/elements/text/display.js [hunk 2]
```diff
@@ -227,6 +226,17 @@ function TextDisplay({
...
 
+  borderRadius =
+    previewMode && borderRadius
+      ? {
+          ...borderRadius,
+          topLeft: dataToEditorX(borderRadius.topLeft),
+          topRight: dataToEditorX(borderRadius.topRight),
+          bottomLeft: dataToEditorX(borderRadius.bottomLeft),
+          bottomRight: dataToEditorX(borderRadius.bottomRight),
+        }
```
Comment (barklund) [inline assets/src/edit-story/elements/text/display.js]: This is now duplicated three times. Also, what's left in the `borderRadius` object to spread in line 232?

**Review by barklund — COMMENTED** [review]
I think the original ticket in #5530 talks about a difference between editor and frontend/amp-output/web-story and not between canvas and preview, though it is using the term "preview" - that is however what the button says when you click to open the amp output.

This work here also seems to be relevant, but I'm not sure it's directly addressing the original ticket.

**Comment by miina** [comment]
@barklund I'm merging this into #5529 since it doesn't really address the intended issue at all, most of it should be in 5529 anyway.

**Pull request merged**